src/main/java/uk/gov/hmcts/divorce/solicitor/service/task/SetConfirmServiceDueDate.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.divorce.solicitor.service.task;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.sdk.api.CaseDetails;
+import uk.gov.hmcts.divorce.divorcecase.model.CaseData;
+import uk.gov.hmcts.divorce.divorcecase.model.State;
+import uk.gov.hmcts.divorce.divorcecase.task.CaseTask;
+
+import java.time.Clock;
+import java.time.LocalDate;
+
+@Component
+@Slf4j
+public class SetConfirmServiceDueDate implements CaseTask {
```

Comment:
Were you going to add any unit tests for the SolicitorSubmitConfirmService and SetConfirmServiceDueDate?

Reply:
Good point - I will add them now

Reply:
Tests added - PR branch rebuilding....